src/main/java/bio/terra/service/tabulardata/google/BigQueryPdao.java

```diff
@@ -555,7 +584,7 @@ public void snapshotViewCreation(
           + ") AS T";
 
   private static final String validateRowIdsForRootTemplate =
-      "SELECT COUNT(1) FROM `<datasetProject>.<dataset>.<table>` AS T, "
+      "SELECT COUNT(1) FROM (<liveViewSql>) AS T, "
```

Comment:
Wouldn't this switch from querying the dataset to the snapshot/view? Is that intended? The variable name insinuates this is being used for validation of the row ids, so it might need to be selected from the source dataset rather than the live view.

Reply:
No, the `liveViewSql` is the live view for the dataset.  Though fair point that this variable isn't well named. Let me rename this one